docs/known-issues.md

```diff
@@ -1,14 +1,14 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+A known issue is any major user-facing bug or lack of user-facing feature that:
+
+* Diverges from vanilla Unity ECS design or implementation. 
+<br/>**OR**
+* Diverges from user expectations of a SpatialOS project (for example; interacting across worker instance boundaries).
```

Comment:
I still think this definition isn't correct 🤷‍♂️

Reply:
I feel like 'known issue' speaks for itself... maybe we could look at changing this to something like: 

>This page outlines known issues that may be fixed in later releases of the GDK. Use this page to see if we are already aware of an issue that may be affecting you. If your issue is not listed here, you can report a bug [here](bug report link)

Or something like that. At the moment it looks like we're trying to redefine the widely known definition of 'known issue'.

Reply:
I really like that version! definitely feels more appropriate than trying to define what a known issue is

Reply:
No idea who wrote this or when. Things just appear in the docs by magic. <"Expecto words!">
I don't know if it was in response to someone's confusion or a request from on high? Checking the blame and seeing who and why.